### ISSUE TITLE
Unexpected output when using --quiet

### ISSUE THREAD

**Issue by na--** [loadimpact/k6#782]
If I execute `k6 --quiet run script.js > output-stdout.log 2> output-stderr.log` where `script.js` contains:
```js
export default function () {
	console.log(`VU ${__VU} executing iteration ${__ITER}`);
}
```

I'd expect to see something like this in one of the files:
```
VU 1 executing iteration 0
```
indeed, `output-stdderr.log` contains the logged message:
```
time="2018-09-27T15:47:52+03:00" level=info msg="VU 1 executing iteration 0"
```
the log format change is somewhat understandable (since we redirect the err stream to a non-TTY output), and even though logging user `console.log()` messages to stderr is a bit strange, at least it's consistent.

My troubles are with `output-stdout.log`, because it contains this:
```

          /\      |‾‾|  /‾‾/  /‾/   
     /\  /  \     |  |_/  /  / /   
    /  \/    \    |      |  /  ‾‾\  
   /          \   |  |‾\  \ | (_) | 
  / __________ \  |__|  \__\ \___/ .io

    init [----------------------------------------------------------] runner
    init [----------------------------------------------------------] options
    init [----------------------------------------------------------] executor
    init [----------------------------------------------------------]   engine
    init [----------------------------------------------------------]   collector
    init [----------------------------------------------------------]   server
  execution: local
     output: -
     script: so.js

    duration: -, iterations: 1
         vus: 1, max: 1

    init [----------------------------------------------------------] starting

```

Which is not very quiet at all...

**Comment by openmohan** [comment]
Just for my understanding:
so when `k6 --quiet run script.js > output-stdout.log 2> output-stderr.log` is ran, the `output-stdout.log` should be populated with console.log message ie : `time="2018-09-27T15:47:52+03:00" level=info msg="VU 1 executing iteration 0"`

Can you please confirm @na--

**Comment by na--** [comment]
@openmohan, not exactly. In my opinion, when `--quiet` mode is enabled, those things shouldn't be outputted at all, regardless if it's on `stderr` or `stdout`.

**Comment by apmattil** [comment]
I would expect the report at stdout even the quiet is on.
 -q, --quiet              disable progress updates
I assumed the progress bar is not shown.. not the whole report.

**Comment by MStoykov** [comment]
@apmattil Yes.
What we actually want is for the whole banner(k6 logo thing), the number of VUs, duration and such information that is printed about the test to be missing + the progressbar. 
But we would like to see all the output of `console.log` from inside the script, all warning/errors that could happen (like the one about the [not enough VUs for all iterations](https://github.com/loadimpact/k6/blob/master/cmd/run.go#L157)) and the end of run summary. The `console.log`  has an [issue](https://github.com/loadimpact/k6/issues/831) and [pr](https://github.com/loadimpact/k6/pull/833) to be able to be redirected, the summary have a way to be removed. Maybe in the future we should have a way for the errors and warning to be hidden as well but this probably should be done in another issue.

**Comment by MStoykov** [comment]
This is the current status of this. But I am starting to question if what we want is for everything to be gone ... after all `-q` does say only "progress updates" and arguably that is the most useful part for most people - not to have a lot of spamming progress updates when running it in CI. 
The summary can be dropped on it's own and we can probably add options for the others as well. 
On the other hand there should probably be one option which does ... all of that and `--quiet` seems like the best choice. 
![image](https://user-images.githubusercontent.com/312246/97850365-9ca00080-1cfc-11eb-99f6-7cabbd1b9ab6.png)

**Comment by tlinhart** [comment]
It would be handy to get rid of all the output. Suppose I want to pipe the JSON output to `jq` to get the metrics and then pipe it straight to `curl` for indexing to Elasticsearch. At the moment, I have to store the output in a file, run `jq` processing at the end of the load test and then index it as a third step.

**Comment by na--** [comment]
This depends on https://github.com/loadimpact/k6/issues/1376
